doc/man3/flux_request_decode.adoc

```diff
@@ -22,7 +22,7 @@ SYNOPSIS
 
  int flux_request_decode_raw (const flux_msg_t *msg,
                               const char **topic,
-                              void *data, int *len);
+                              void **data, int *len);
```

Comment:
I may be so very wrong about wanting to do this, but it was quite on purpose, and the reason the result parameter is defined as a `void *` instead of a `void **` is so that the user doesn't have to do something like
```c
const char *data;
int len;

flux_request_decode_raw (msg, NULL, (void **)&data, &len);
```
in virtually every place this function is used.